e2e/cypress/integration/integrations/builtin_commands/user_status_commands_spec.js

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+const testCases = [
+    {command: '/away', ariaLabel: 'Away Icon', message: 'You are now away (Only visible to you)'},
+    {command: '/dnd', ariaLabel: 'Do Not Disturb Icon', message: 'Do Not Disturb is enabled. You will not receive desktop or mobile push notifications until Do Not Disturb is turned off. (Only visible to you)'},
+    {command: '/offline', ariaLabel: 'Offline Icon', message: 'You are now offline (Only visible to you)'},
+    {command: '/online', ariaLabel: 'Online Icon', message: 'You are now online (Only visible to you)'},
+];
+
+describe('I18456 Built-in slash commands: user status via post', () => {
+    before(() => {
+        // # Login as user-1, go to "/" and set user status to online
+        cy.apiLogin('user-1');
+        cy.visit('/');
+        cy.apiUpdateUserStatus('online');
+    });
+
+    after(() => {
+        // # Set user status to offline
+        cy.apiUpdateUserStatus('online');
+    });
+
+    testCases.forEach((testCase) => {
+        it(testCase.command, () => {
+            // # Post slash command
+            cy.postMessage(testCase.command + ' ');
+
+            verifyUserStatus(testCase);
+        });
+    });
+});
+
+describe('I18456 Built-in slash commands: user status via suggestion list', () => {
+    before(() => {
+        // # Login as user-1, go to "/" and set user status to online
+        cy.apiLogin('user-1');
+        cy.visit('/');
+        cy.apiUpdateUserStatus('online');
+    });
+
+    beforeEach(() => {
+        // # Type "/" on textbox
+        cy.get('#post_textbox').clear().type('/');
+    });
+
+    after(() => {
+        // # Set user status to offline
+        cy.apiUpdateUserStatus('online');
+    });
+
+    it('/away', () => {
+        const testCase = testCases[0];
+
+        // # Verify that the suggestion list is visible
+        cy.get('#suggestionList').should('be.visible').then((container) => {
+            // # Find command and click
+            cy.getByText(/\/away/, {container}).click({force: true});
+        });
+
+        cy.get('#post_textbox').type(' {enter}');
+
+        verifyUserStatus(testCase);
+    });
+
+    it('/dnd', () => {
```

Comment:
Any specific reason why we are not iterating through the testcases like you did above. 

Something like this:
`
    testCases.forEach((testCase) => {
        it(testCase.command, () => {
            // # Post slash command
            cy.postMessage(testCase.command + ' ');

            verifyUserStatus(testCase);
        });
    });
`

Reply:
Purpose of this test is selecting thru suggestion list where it's not working with the regex in getting by text (e.g. `cy.getByText(/\/dnd/, {container})`